### ISSUE TITLE
Register GHG source_id and institution_id

### ISSUE THREAD

**Issue by znichollscr** [PCMDI/input4MIPs_CVs#13]
@durack1 moving discussion from #12 here.

Summary of @durack1's comment [here](https://github.com/PCMDI/input4MIPs_CVs/pull/12#issuecomment-2048107225):

- excel spreadsheet where attributes was discussed is here: https://docs.google.com/spreadsheets/d/1vMlMVAnP25GpHcS0B8cyvFvUoTZxlUvxBmFB-PbLDG0/edit#gid=108565686
- idea is to move as much info as possible into the source ID JSON file in this repo, with the suggestion being as below

```json
        "PCMDI-AMIP-1-1-9":{
            "calendar":"gregorian",
            "comment":"Based on Hurrell SST/sea ice consistency criteria applied to merged HadISST (1870-01 to 1981-10) & NCEP-0I2 (1981-11 to 2022-12)",
            "contact":"PCMDI (pcmdi-cmip@llnl.gov)",
            "dataset_category":"SSTsAndSeaIce",
            "further_info_url":"https://pcmdi.llnl.gov/mips/amip",
            "grid":"1x1 degree longitude x latitude",
            "grid_label":"gn",
            "institution":"Program for Climate Model Diagnosis and Intercomparison, Lawrence Livermore National Laboratory, Livermore, CA 94550, USA",
            "institution_id":"PCMDI",
            "license":"AMIP boundary condition data produced by PCMDI is licensed under a Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses/by/4.0). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing input4MIPs output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file). The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law",
            "mip_era":"CMIP6Plus",
            "nominal_resolution":"1x1 degree",
            "product":"observations",
            "references":"Taylor, K.E., D. Williamson and F. Zwiers, 2000: The sea surface temperature and sea ice concentration boundary conditions for AMIP II simulations. PCMDI Report 60, Program for Climate Model Diagnosis and Intercomparison, Lawrence Livermore National Laboratory, 25 pp. Available online: https://pcmdi.llnl.gov/report/pdf/60.pdf",
            "region":[
                "global_ocean"
            ],
            "release_year":"2023",
            "source":"PCMDI-AMIP 1.1.9: Merged SST based on UK MetOffice HadISST and NCEP OI2",
            "source_description":"Sea surface temperature and sea-ice datasets produced by PCMDI (LLNL) for the AMIP (DECK) experiment of CMIP6Plus",
            "source_id":"PCMDI-AMIP-1-1-9",
            "source_type":"satellite_blended",
            "source_variables":[
                "areacello",
                "sftof",
                "siconc",
                "siconcbcs",
                "tos",
                "tosbcs"
            ],
            "source_version":"1.1.9",
            "target_mip":"CMIP",
            "title":"PCMDI-AMIP 1.1.9 dataset prepared for input4MIPs"
        },
```

Summary of my reply [here](https://github.com/PCMDI/input4MIPs_CVs/pull/12#issuecomment-2048904169):

- I think I understand
- Does it make sense to remove some of the data file specific keys (e.g. grid) to avoid having to make new source IDs for each data file? That would mean the source ID info could be applied to all data files in a data set, with some data file level metadata being captured elsewhere (probably in the file I guess)

Discussion can continue below.

**Comment by durack1** [comment]
We'll need to register `Climate Resource` here, who already have an ROR - https://ror.org/01kk86953

**Comment by znichollscr** [comment]
xref #8, do we register here or in https://github.com/PCMDI/input4MIPs-cmor-tables

**Comment by durack1** [comment]
> xref #8, do we register here or in https://github.com/PCMDI/input4MIPs-cmor-tables

@znichollscr `input4MIPs-cmor-tables` is circa CMIP6, whereas I am trying to pull anything post-CMIP6 (CMIP6Plus, CMIP7, ...) forward in time to this repo

**Comment by durack1** [comment]
And attempting to continue the discussion as I have closed #12.

The `source_id` recommended here is `CR-CMIP-0-1-0`, as the CMIP DECK is identified as `CMIP` activity_id across CMIP6, and that will likely continue going forward.

Registering multiple source_id's are a to-do imminently, so will add the `CR-CMIP-0-1-0` to the list if you approve this identity @znichollscr - we'll need to also capture additional dataset metadata, ala following the template for `PCMDI-AMIP-1-1-9` immediately above https://github.com/PCMDI/input4MIPs_CVs/issues/13#issue-2236887835

**Comment by znichollscr** [comment]
Sounds good, CR-CMIP-0-1-0 is fine
